v-next/hardhat/templates/typescript-with-examples/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "typescript-with-examples",
+  "description": "A TypeScrpit Hardhat project with examples",
+  "type": "module",
+  "devDependencies": {
+    "@ignored/hardhat-vnext": "workspace:^3.0.0-next.3",
+    "@ignored/hardhat-vnext-node-test-runner": "workspace:^3.0.0-next.2",
```

Comment:
A project won't be able to initialize from this template right now because we haven't published `hardhat-vnext-node-test-runner` yet.